src/components/section/MobileHeader.jsx

```
@@ -0,0 +1,151 @@
+              type='search'
+              placeholder='검색어를 입력해주세요.'
+              autoComplete='off'
+              className='mobile-search-input'
+              onChange={e => setSearchInput(e.target.value)}
+              onKeyDown={e => {
+                if (e.key === 'Enter') {
+                  handleSearch();
+                }
+              }}
+            />
+            <button onClick={handleSearch}><IoIosSearch /></button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)